artiq/frontend/artiq_client.py

```diff
@@ -80,10 +81,12 @@ def get_argparser():
                                     help="name of the dataset")
     parser_set_dataset.add_argument("value", metavar="VALUE",
                                     help="value in PYON format")
-    parser_set_dataset.add_argument("-p", "--persist", action="store_true",
-                                    help="make the dataset persistent")
-    parser_set_dataset.add_argument("-n", "--no-persist", action="store_true",
-                                    help="make the dataset non-persistent")
+    # Allow only one of --persist or --no-persist arguments
```

Comment:
This is already pretty clear from the name of the function, ``add_mutually_exclusive_group``.

Reply:
Ok. So it is an extra comment to make things just slightly more explicit.

Reply:
Resolved in https://github.com/m-labs/artiq/pull/1185/commits/01e4d70dc1e7b2df035382a3a44a3a064999d096